scripts/upgrade.sh

```diff
@@ -10,6 +10,9 @@ if [ $? -ne 0 ]; then
     exit 1
 fi
 
+# Clean mongodb data for legacy nodes
+docker volume rm prover-node-docker_mongodb_data &> /dev/null
```

Comment:
Put it before line 5.
This is special as in fact, currently merkle db had been about  60GB because of client's "copy inherit" tests.